cpu/stm32_common/include/periph_cpu_common.h

```diff
@@ -43,10 +43,26 @@ extern "C" {
  * @brief   Available peripheral buses
  */
 enum {
```

Comment:
Can it also be guarded with `#ifndef HAS_CLOCK_TREE_T` that way we can prevent accidentally setting APB2 clocks in the F0 for example :)